packages/tc-ui/README.md

```diff
@@ -50,9 +50,16 @@ Base url for the treecreeper REST api
 
 Base url for the treecreeper GraphQL api
 
+##### clientId
+
+String to send to the API as the `Client-Id` header
+
 ##### apiHeaders
 
-Function that is passed the object received as input, and should return an object containing the headers to use when calling the treecreeper endpoints
+Either:
+
+-   Function that is passed the object received as input, and should return an object containing additional headers to use when calling the treecreeper endpoints e.g. `api-key`, `client-user-id`.
```

Comment:
is `client-user-id` supposed to be `client-id`? if not, what is the difference?

Reply:
Here's a documentation of what the value contains https://github.com/Financial-Times/treecreeper/blob/master/packages/tc-api-rest-handlers/README.md#metadata (although by the time it gets to this point in the app it is already converted from a header to a property of an object).

I will change the names above to Api-Key and Client-User-Id though as lowercasing them here is inconsistent with headers mentioned higher up